### DISCUSSION TITLE
حذف تغیرنام کانفیگ

### DISCUSSION THREAD

**Discussion by shuvvan** [hiddify/Hiddify-Manager#535] [Bug باگ: please use issue]
سلام و خسته نباشید به تمام زحمتکشان این پنل !

چرا تغییر نام کانفیگ ها حذف شده؟ الان من واسه تست یه تغییر نام الکی داده بودم الان چطوری درستشون کنم ؟

اون گزینه خیلی کاربردی بود لطفا برشگردونید

**Comment by peymanch** [comment]
بله لطفا" این قابلیت رو برگردونید خیلی کاربردی و لازم بود
